backend/app/store/engine/engine.go

```diff
@@ -25,6 +25,7 @@ type Interface interface {
 	Delete(req DeleteRequest) error                             // delete post(s) by id or by userID
 	Flag(req FlagRequest) (bool, error)                         // set and get flags
 	ListFlags(req FlagRequest) ([]interface{}, error)           // get list of flagged keys, like blocked & verified user
+	UserDetail(req UserDetailRequest) (string, error)           // set and get user details
```

Comment:
Because of `UserDetail` storing strings, in order to use it for storing a list of subscriptions, we would need to store them as a comma-separated string.
Subscription per-thread right now is not supported, entering user email subscribes him to every response to his or her comments.